app/views/challenges/show.html.slim

```diff
@@ -80,23 +80,25 @@
                         td = html_escape(badgemap.description).html_safe
 
         .col-lg-6
-          - if logged_in? && (current_user.is_moderator || current_user.is_admin)
+          - if logged_in_as_moderator && @challenge.id != 1 && !@challenge.seasonal
```

Comment:
Use `logged_in_as_moderator?`. Also, I object to the magic `!= 1`. Try adding a `@challenge.site_challenge?` (or whatever you'd call the DED + seasonal challenges) encompassing this check.